contracts/libs/0.8.x/SplitFundsLib.sol

```diff
@@ -0,0 +1,275 @@
+// SPDX-License-Identifier: LGPL-3.0-only
+// Created By: Art Blocks Inc.
+
+import "../../interfaces/0.8.x/IMinterBaseV0.sol";
+import "../../interfaces/0.8.x/IGenArt721CoreContractV3_Base.sol";
+import "../../interfaces/0.8.x/IGenArt721CoreContractV3.sol";
+import "../../interfaces/0.8.x/IGenArt721CoreContractV3_Engine.sol";
+
+import "@openzeppelin-4.7/contracts/token/ERC20/IERC20.sol";
+
+pragma solidity ^0.8.0;
+
+/**
+ * @title Art Blocks Split Funds Library
+ * @notice [TO FILL OUT]
```

Comment:
When we fill this out, lets make sure we also have natspec comments for the `ProjectConfig` struct and all our methods here as well (ChatGPT is great at writing initial comments btw hah 😄).